test/common-test.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
         var graph = new noflo.Graph();
         _.each(componentModules, function(module, name) {
```

Comment:
Can we use a more descriptive name than "name" ?  This is the component name, yes?   Can we call this something like "componentName"?